include/rfb.js

```diff
@@ -1120,7 +1120,7 @@ var RFB;
 
             switch (xvp_msg) {
                 case 0:  // XVP_FAIL
-                    this._updateState(this._rfb_state, "Operation Failed");
+                    Util.Error("Operation Failed");
```

Comment:
Should there be a way to message the UI that this occurred?  It seems like this might be useful information for the user to know (and you can't assume that logging communicates anything to the user).